### PR TITLE
Update ug page

### PR DESCRIPTION
**Pull request by glozxi**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -41,153 +50,116 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+[Back to top](#user-guide-for-taskgenie)
```
Comment (ExtraShotLatte) [inline docs/UserGuide.md:165]: Love this inclusion of back to top!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -41,153 +50,116 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-
---------------------------------------------------------------------------------------------------------------------
+Paidlancer data are saved in the hard disk automatically exiting the program. There is no need to save manually.
 
 ## FAQ
 
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**Q**: How do I transfer my data to another Computer?  
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous TaskGenie home folder.
```
Comment (ExtraShotLatte) [inline docs/UserGuide.md:153]: TaskGenie? @engenhui1999 Let's change this to PaidLancers!

**Review by ExtraShotLatte — CHANGES_REQUESTED** [review]
LGTM, some small nits on the consistency of our naming for the application `PaidLancers` to be reflected properly in the UG before we merge this please.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -41,153 +50,116 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
-
+### Add a Contact
+Adding a contact : `newcontact`
+Adding a contact to contacts
+Format: newcontact NAME NUMBER
+Creates a new contact with specified NAME and NUMBER
+Example:
+newcontact Deborah Tan /num 91234567
```
Comment (ExtraShotLatte) [inline docs/UserGuide.md:59]: Is our syntax `newcontact /n <name> /num <hp no>` or is the /n no needed?

File: docs/UserGuide.md (hunk 2 above)
Comment (glozxi) [reply docs/UserGuide.md:153]: To make sure, is it "PaidLancers" or "Paidlancers"?

Comment (ExtraShotLatte) [reply docs/UserGuide.md:153]: My bad, `Paidlancers` according to our documents. Thank you for pointing that out!

File: docs/UserGuide.md (hunk 3 above)
Comment (bryansendeavour) [reply docs/UserGuide.md:59]: Currently the revised syntax I have designed is `newcontact /n NAME /num NUMBER` , this is to facilitate the ease of identifying the NAME and NUMBER. This also compliments the way our skeleton code works.

**Review by EksdeeeX — APPROVED** [review]
LGTM

**Review by ExtraShotLatte — APPROVED** [review]
LGTM TOO!

**Pull request merged**